ExpressionFramework/Classes/Expression Framework/Internal Interfaces/Cocoa Touch Extensions/Foundation Extensions/Date+Extensions.swift

```diff
@@ -10,32 +10,23 @@ import Foundation
 
 extension Date {
 
-    private static let dateStampFormatter:DateFormatter = {
+    var timeStamp: String {
```

Comment:
date Formatters are very expensive to instantiate, therefore recommendation is to keep them in memory